src/renderer/components/ui/activities/begin/games/MyCracha.vue

```diff
@@ -1,33 +1,53 @@
+            }"
+        >
             <b-col v-if="hasKeys" class="activity-keys">
                 <b-row>
-                    <b-col :sm="keyColSize" class="imagem"> 
+                    <b-col :sm="keyColSize" class="imagem">
                         <div class="item">
-                            <ls-card-input-cracha 
-                                class="cracha" 
-                            >
+                            <ls-card-input-cracha class="cracha">
                                 <div class="outer">
-                                    <div class="card-input card--input-image">
+                                    <div
+                                        class="card-input card--input-image"
+                                        :class="{ 'no-photo-warning': hasNoImage }"
+                                    >
                                         <div class="icon-photo">
-                                            <input 
+                                            <input
                                                 :ref="`file${name.text}`"
-                                                style="display: none"                                             
-                                                type="file"     
-                                                accept="image/gif, image/jpeg, image/png"                                         
-                                                @change="handleFileUpload(name.text)"                 
-                                            >
+                                                style="display: none"
+                                                type="file"
+                                                accept="image/gif, image/jpeg, image/png"
+                                                @change="handleFileUpload(name.text)"
+                                            />
                                         </div>
                                         <div v-if="userName !== null" :ref="'imgContainer'" class="image">
-                                            <div :style="{backgroundImage: `url('${imgSrc}')`}" class="photo" />
+                                            <div
+                                                :style="{ backgroundImage: `url('${imgSrc}')` }"
+                                                class="photo"
+                                            />
                                         </div>
                                     </div>
                                 </div>
                                 {{ name.text }}
                             </ls-card-input-cracha>
                         </div>
                     </b-col>
+                    <div>
+                        <div class="btn-container">
+                            <div class="btn-container--border">
+                                <b-button class="btn-continue" @click="continueGame()">
```

Comment:
Também é interessante sumir com o botão enquanto não tiver a foto.
```suggestion
                                <b-button class="btn-continue" v-if="!hasNoImage" @click="continueGame()">
```